src/dynamic.ts

```diff
@@ -30,26 +29,26 @@ export class Dynamic extends Command {
     }
 
     public async run(args: string[]): Promise<Command> {
-            var args2 = args.slice(2);
-            var bigArgs = [];
-            var indexes = [];
+            var parsedArgs = args.slice(2);
+            var commandArrays = [];
+            var commandIndex = [];
             try {
-                for(let i=0;i<args2.length;i++){
-                    for(var cmd in COMMAND_POINTER){
-                        if(args2[i] === COMMAND_POINTER[cmd]){
-                            indexes.push(i)
+                for(let i=0;i<parsedArgs.length;i++){
+                    for(var cmd in this.commands){
```

Comment:
Prettier and more consistent. You can follow their lead on looping through the commands.

https://github.com/ictinusinc/graphene-cli/blob/0c88c66b154462c90eaf0af6cf5a13112d858c8f/src/command.ts#L31-L35